haystack/graph_retriever/base.py

```diff
@@ -0,0 +1,16 @@
+from haystack import BaseComponent
+from haystack.knowledge_graph.base import BaseKnowledgeGraph
+
+
+class BaseGraphRetriever(BaseComponent):
+    knowledge_graph: BaseKnowledgeGraph
+    outgoing_edges = 1
+
+    def retrieve(self, question_text: str, top_k: int):
```

Comment:
Should we replace `question_text` with `query`, as that what we use in other components?

Reply:
Yes, I replaced the "question" terminology with "query" for consistency reasons.